src/lib.rs

```diff
@@ -692,6 +718,44 @@ impl From<value::Value> for Value {
     }
 }
 
+#[wasm_bindgen]
+pub struct U128(u128);
+
+impl From<u128> for U128 {
+    fn from(number: u128) -> U128 {
+        U128(number)
+    }
+}
+
+#[wasm_bindgen]
+impl U128 {
+    pub fn from_be_bytes(bytes: Uint8Array) -> Result<U128, JsValue> {
```

Comment:
would 2 u64 be easier ?

Reply:
do you mean a constructor from two u64 like high and low? (that means, two BigInt from javascript) I don't know, maybe, I can change or just add both